src/SFA.DAS.ApplyService.Domain/Apply/Answer.cs

```diff
@@ -16,7 +16,7 @@ public string Value
             set { JsonValue = value; }
         }
 
-        [JsonProperty(PropertyName = "Value")]
+        [JsonProperty(PropertyName = "Status")]
```

Comment:
I think this domain object is used in apply, so this might break serialisation elsewhere??

Reply:
It could be directly SectionReviewStatus

Reply:
That sholudn't have been changed, I must have missed it in my final review before TRing
.  I did a replace value with status in my object, and it must have picked this up.  I'm changing that back.

Reply:
changed back now